account_move_ref_account/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+{
+    'name': 'Account Move Source Document for Invoice/Payment/Statement',
+    'summary': 'Journal Entry with Source Document Link',
+    'version': '12.0.1.0.0',
+    'author': 'Ecosoft,Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/account-financial-tools',
+    'category': 'Accounting & Finance',
+    'depends': ['account_move_ref_document',
+                'account'],
+    'data': [],
```

Comment:
@kittiu If not any `file` then remove `data`

Reply:
Done